### PR TITLE
Split README into smaller parts

### PR DESCRIPTION
**Pull request by dpc**
[Rendered](https://github.com/dpc/minimint/tree/readme-tweaks)

See #404

### CODE REVIEW

**Comment by justinmoon** [comment]
I think this is a better structure. It could use a little more content but we can do that over time. 

Thoughts @elsirion @jkitman?

**Comment by dpc** [comment]
> I think this is a better structure. It could use a little more content but we can do that over time.

Agreed. It looks a bit short now, but that's mostly because there's still plenty of pages to write.

**Review by NicolaLS — APPROVED** [review]
I also think this is a better structure (especially later on), lgtm

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Contributions are very welcome, just open an issue or PR if you see something to improve! See our other [communication channels](../README.md#getting-involved) above to coordinate with other contributors!
+
+Please note that all contributions happen under the MIT license as described at the bottom of this page.
+
+## Getting involved
+
+If you want to learn more about the general idea go to [fedimint.org](https://fedimint.org/), beyond a high level explanation you can also find links to talks and blog posts at the bottom of the page.
+
+Visit our
```
Comment (okjodom) [inline docs/contributing.md:11]: This is a dangling reference yet I still really want to visit wherever it goes! 😎

**Review by okjodom — COMMENTED** [review]
🔥🔥🎇

**Comment by jkitman** [comment]
Looks good!  Perhaps we should also add links to how to run the integration tests, the website and architecture docs.

Once we have a production-ready release then instructions for running it should probably live in the README as well to be easily accessible.

File: docs/contributing.md (hunk 1 above)
Comment (dpc) [reply docs/contributing.md]: :eyes: Fixed.

**Review by okjodom — APPROVED** [review]

**Comment by dpc** [comment]
Can we land? It doesn't look like anyone is opposed.

**Review by justinmoon — APPROVED** [review]

**Pull request merged**